src/token/api/src/tx_record.rs

```diff
@@ -2,10 +2,10 @@ use candid::{CandidType, Deserialize, Principal};
 use ic_canister::ic_kit::ic;
 use ic_helpers::tokens::Tokens128;
 
-use crate::account::{Account, AccountInternal};
-use crate::types::{Operation, Timestamp, TransactionStatus, TxId};
-
-use super::Memo;
+use crate::{
+    account::{Account, AccountInternal},
+    types::{Memo, Operation, Timestamp, TransactionStatus, TxId},
```

Comment:
`TxId` can probably live with the `TxRecord`

Reply:
Yeah, true story 👍🏾